keps/sig-auth/3331-structured-authentication-configuration/README.md

```diff
@@ -739,49 +650,90 @@ providers such as Okta, Azure AD, etc:
+- Add tests
+  - Tests for automatic reload of the configuration
+
+#### Pre-GA follow-up
+
+- With automatic reload of configuration, typo in issuer URL will not be detected since an issuer is explicitly allowed to be offline when an API server is starting up to allow for self-hosted IDPs. We need
+  to come-up with an approach to make this more robust.
 
 #### GA
 
+- Gather feedback
 - Add a full documentation with examples for the most popular providers, e.g., Okta, Dex, Auth0
 - Migration guide
-- Deprecation warnings for non-structured OIDC provider configuration
+- e2e test with an external provider completed and enabled
+- Get distributed claims working with CEL
+- Decide if we want to support egress selection configuration and how to do so
 
-#### Deprecation
+### Upgrade / Downgrade Strategy
 
-kube-apiserver `--oidc-*` flags require deprecation warnings on the stable release of the feature.
+While the feature is in Alpha, there is no change if cluster administrators want to
+keep on using command line flags.
 
-### Upgrade / Downgrade Strategy
+When the feature goes to Beta/GA or the cluster administrators want to configure
+jwt authenticators using the configuration file, they need to make sure:
 
-<!--
-If applicable, how will the component be upgraded and downgraded? Make sure
-this is in the test plan.
-
-Consider the following in developing an upgrade/downgrade strategy for this
-enhancement:
-- What changes (in invocations, configurations, API use, etc.) is an existing
-  cluster required to make on upgrade, in order to maintain previous behavior?
-- What changes (in invocations, configurations, API use, etc.) is an existing
-  cluster required to make on upgrade, in order to make use of the enhancement?
--->
+1. The configuration file is available on the API server and the `--authentication-config` flag is set.
+2. No `--oidc-*` flags are set.
+
+When downgrading from the structured configuration to the flag-based configuration, they need to
+unset the `--authentication-config` flag and restore the `--oidc-*` flags to configure the JWT authenticator.
 
 ### Version Skew Strategy
 
-<!--
-If applicable, how will the component handle version skew with other
-components? What are the guarantees? Make sure this is in the test plan.
-
-Consider the following in developing a version skew strategy for this
-enhancement:
-- Does this enhancement involve coordinating behavior in the control plane and
-  in the kubelet? How does an n-2 kubelet without this feature available behave
-  when this feature is used?
-- Will any other components on the node change? For example, changes to CSI,
-  CRI or CNI may require updating that component before the kubelet.
--->
+This is an API server only change and does not affect other components. If the API server is
+not the minimum required version (v1.29), the feature will not be available.
+
+<<[UNRESOLVED open questions that don't clearly fit elsewhere ]>>
+## Open Questions
```

Comment:
Add some verbiage to indicate that these must be addressed or rejected or deferred before this KEP is marked as GA.